src/commands/build.js

```diff
@@ -2,12 +2,13 @@ import fs from "fs-extra";
 import path from "path";
 import webpack from "webpack";
 import getWebpackConfig from "../config/getWebpackClientConfig";
+import generateStaticHTMLFile from "../lib/server/generateStaticHTMLFile";
```

Comment:
@toddw `gluestick run` and `gluestick build` are currently broken on this branch. If I remove this import statement they work. I narrowed it down to lines [30-32](https://github.com/TrueCar/gluestick/blob/328845bc516f0df4ceec2a0bcd41a14401d8d7bf/src/lib/server/RequestHandler.js#L30) in RequestHandler.js. If I remove that block, `build` and `run` work (presumably because it's set to the same thing on line [134](https://github.com/TrueCar/gluestick/blob/328845bc516f0df4ceec2a0bcd41a14401d8d7bf/src/lib/server/RequestHandler.js#L134) if null?), but then `build --static` throws the same error that `build` and `run` were. I've tried a few things, but I think I'm stuck, at least as far as this piece is concerned.

```
gluestick build --static                                                                                              ⏎
/Users/marc/Code/static-gluestick/node_modules/react-transform-hmr/lib/index.js:51
    throw new Error('locals[0] does not appear to be a `module` object with Hot Module ' + 'replacement API enabled. You should disable react-transform-hmr in ' + 'production by using `env` section in Babel configuration. See the ' + 'example in README: https://github.com/gaearon/react-transform-hmr');
    ^

Error: locals[0] does not appear to be a `module` object with Hot Module replacement API enabled. You should disable react-transform-hmr in production by using `env` section in Babel configuration. See the example in README: https://github.com/gaearon/react-transform-hmr
    at proxyReactComponents (/Users/marc/Code/static-gluestick/node_modules/react-transform-hmr/lib/index.js:51:11)
    at Object.<anonymous> (/Users/marc/Code/static-gluestick/src/config/.entry.js:59:59)
    at Module._compile (module.js:556:32)
    at loader (/Users/marc/Code/marcdel/gluestick/node_modules/babel-core/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/marc/Code/marcdel/gluestick/node_modules/babel-core/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
```